data-raw/healthy-places/access-to-green-space/access-greenspace.R

```diff
@@ -0,0 +1,30 @@
+# Load packages
+library(tidyverse)
+library(geographr)
+
+# Load lookup dataset
+lookup_ltla_raw <- lookup_ltla_ltla
+
+# Filter the dataset for Welsh local authorities - LTLA code and LTLA name
+raw_welsh_greenspace <- data.frame ()
```

Comment:
`raw` is typically used as part of a variable name when you are loading a dataset raw, i.e. before it is tidied and clean. In this instance you are creating a new dataframe. See this --> http://adv-r.had.co.nz/Style.html#:~:text=Generally%2C%20variable%20names%20should%20be,this%20is%20not%20easy!).&text=Where%20possible%2C%20avoid%20using%20names,the%20readers%20of%20your%20code. on how to name objects, variables and functions. Note that this whole repo is about Wales so having `welsh` in the variable name may not be a good differentiator.

Reply:
I changed it to "cleaned_greenspace"